scripts/image-builder/build.sh

```diff
@@ -336,23 +341,41 @@ if [ -z "${IMAGE_VERSION}" ] ; then
     exit 1
 fi
 
-# Download openshift local RPM packages (noarch for python and selinux packages)
-rm -rf openshift-local 2>/dev/null || true
-
-OCP_REPO_NAME="rhocp-4.13-for-rhel-${OSVERSION}-$(uname -m)-rpms"
-reposync -n -a "${BUILD_ARCH}" -a noarch --download-path openshift-local \
-    --repo="${OCP_REPO_NAME}" \
-    --repo="fast-datapath-for-rhel-${OSVERSION}-${BUILD_ARCH}-rpms" >/dev/null
+title "Ensure there is at least one osbuild worker running"
+sudo systemctl start osbuild-composer.socket
+sudo systemctl start "osbuild-worker@1.service"
+
+# Enable rhcop-4.13 repo for dependencies
+title "Enabling RHOCP and fast-datapath repositories"
+cat >rhocp-4.13.toml <<EOF
+id = "rhocp-4.13"
+name = "Red Hat OpenShift Container Platform 4.13 for RHEL ${OSVERSION}"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel${OSVERSION}/$(uname -m)/rhocp/4.13/os"
+check_gpg = true
+check_ssl = true
+system = false
+rhsm = true
+EOF
+sudo composer-cli sources add rhocp-4.13.toml
+
+cat >fast-datapath.toml <<EOF
+id = "fast-datapath"
+name = "Fast Datapath for RHEL ${OSVERSION}"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel${OSVERSION}/$(uname -m)/fast-datapath/os"
+check_gpg = true
+check_ssl = true
+system = false
+rhsm = true
+EOF
+sudo composer-cli sources add fast-datapath.toml
```

Comment:
I think this code is best converted to .toml.template and expanded in the loop at [ line399](https://github.com/openshift/microshift/pull/2028/files#diff-bb6dd0287122cf14166d793e40a00a53e4cf43748f2318fad274ff4028020906R399)

In any case, we should first delete the source and then re-add it. Otherwise, fix of a toml does not get into osbuilder.